alembic/versions/307a4c43ae47_add_success_to_srpmbuildmodel.py

```diff
@@ -0,0 +1,28 @@
+"""Add 'success' to SRPMBuildModel
+
+Revision ID: 307a4c43ae47
+Revises: c6250555a36c
+Create Date: 2020-04-15 11:07:32.488213
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = "307a4c43ae47"
+down_revision = "c6250555a36c"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column("srpm_builds", sa.Column("success", sa.Boolean(), nullable=True))
```

Comment:
this indicate whether the srpm was created or not, right?

Reply:
It indicates if the `srpm` build went well.
- `False` = there was an error
- `True` = the `srpm` was created without error